src/main/scala/rocketchip/Periphery.scala

```diff
@@ -17,6 +17,11 @@ import rocket.XLen
 import scala.math.max
 import coreplex._
 
+case object MIFAddrBits extends Field[Int]
+case object MIFDataBits extends Field[Int]
+case object MIFTagBits extends Field[Int]
+case object MIFDataBeats extends Field[Int]
+
```

Comment:
So I think we actually still need an ExtMMIODataBits. This is just for the MMIO ports that we export off chip. It's actually possible that we will need different widths for each exported MMIO port. But for now, let's just have one.